packages/plugin-session-replay/rollup.config.js

```diff
@@ -0,0 +1,6 @@
+import { iife, umd } from '../../scripts/build/rollup.config';
+
+iife.input = umd.input;
+iife.output.name = 'sessionReplay';
+
+export default [umd, iife];
```

Comment:
this uploads the bundle to s3 and makes them available through our CDN. can be used by customers installing the SDK via the snippet loader.

Reply:
Awesome!